server/src/test/java/org/elasticsearch/common/io/stream/ByteBufferStreamInputTests.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.common.io.stream;
+
+import org.apache.lucene.util.BytesRef;
+import org.elasticsearch.common.bytes.BytesReference;
+
+import java.io.IOException;
+
+public class ByteBufferStreamInputTests extends AbstractStreamTests {
```

Comment:
It isn't really about the ByteBuffer class, more about BytesRef?

Reply:
I changed to so it is more obvious how the ByteBufferStreamInput is used in the test.